src/ray/protobuf/gcs_service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+import "src/ray/protobuf/gcs.proto";
+
+message AddJobRequest {
+  JobTableData data = 1;
+}
+
+message AddJobReply {
+  bool success = 1;
+}
+
+message MarkJobFinishedRequest {
+  bytes job_id = 1;
+}
+
+message MarkJobFinishedReply {
+  bool success = 1;
+}
+
+// Service for job info access.
+service JobInfoAccessService {
```

Comment:
I feel `Access` is kind of redundant. What about `JobInfoGcsService` and document it as "Part of the `GcsService` that handles job-info-related requests."?

Reply:
Can we have 1 service definition for the GCS server, instead of one for each endpoint?

Reply:
> Can we have 1 service definition for the GCS server, instead of one for each endpoint?

Yes we will integrate all definitions of `XXXService` in a single proto file. But the implementation of each service should be put into a file separately, otherwise, there is too much code in a single file.

Reply:
> I feel `Access` is kind of redundant. What about `JobInfoGcsService` and document it as "Part of the `GcsService` that handles job-info-related requests."?

Okey, As discussed, it will be replaced with `JobInfoGcsService`, meanwhile the `job_info_access_handler` will be replaced with `job_info_handler`.